Misc/NEWS.d/next/C_API/2025-01-15-11-42-07.gh-issue-128863.C9MkB_.rst

```diff
@@ -8,6 +8,14 @@ Python 3.18:
 * :c:func:`!_PyLong_New`: use :c:func:`PyLongWriter_Create`.
 * :c:func:`!_PyThreadState_UncheckedGet`: use :c:func:`PyThreadState_GetUnchecked`.
 * :c:func:`!_PyUnicode_AsString`: use :c:func:`PyUnicode_AsUTF8`.
+* :c:func:`!_PyUnicodeWriter_Init`: use :c:func:`PyUnicodeWriter_Create(0) <PyUnicodeWriter_Create>` instead.
+* :c:func:`!_PyUnicodeWriter_Finish`: use :c:func:`PyUnicodeWriter_Finish`.
+* :c:func:`!_PyUnicodeWriter_Dealloc`: use :c:func:`PyUnicodeWriter_Discard`.
+* :c:func:`!_PyUnicodeWriter_WriteChar`: use :c:func:`PyUnicodeWriter_WriteChar`.
+* :c:func:`!_PyUnicodeWriter_WriteStr`: use :c:func:`PyUnicodeWriter_WriteStr`.
+* :c:func:`!_PyUnicodeWriter_WriteSubstring`: use :c:func:`PyUnicodeWriter_WriteSubstring`.
+* :c:func:`!_PyUnicodeWriter_WriteASCIIString`: use :c:func:`PyUnicodeWriter_WriteUTF8`.
+* :c:func:`!_PyUnicodeWriter_WriteLatin1String`: use :c:func:`PyUnicodeWriter_WriteUTF8`.
```

Comment:
Can you please document how to update the old calls to the new ones? AFAIK, it's only straightforward if you already know what changed.

Reply:
I tried to keep the documentation short, but ok, I added more explicit examples such as:

> _PyUnicodeWriter_WriteASCIIString(): replace _PyUnicodeWriter_WriteASCIIString(&writer, str) with PyUnicodeWriter_WriteUTF8(writer, str).